jte-runtime/src/main/java/gg/jte/runtime/StringUtils.java

```diff
@@ -70,4 +70,22 @@ public static void appendEscaped(StringBuilder result, String string) {
             }
         }
     }
+
```

Comment:
Can you please move these methods to a StringUtils class in the JSP converter project? It is not required by jte itself.

I know we can't use `String.repeat`, because of Java 8 compatibility.